src/RageFileManager.h

```diff
@@ -4,6 +4,7 @@
 #include <unordered_set>
 #include <vector>
 
+#include "global.h"
```

Comment:
one day ill also go through and iwyu but that sounds like an annoying effort in the short term